test/api/v3/integration/groups/POST-groups_groupId_leave.js

```diff
@@ -0,0 +1,86 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
+
+describe('POST /groups/:groupId/leave', () => {
+  let user;
+  let userToInvite;
+  let group;
```

Comment:
Well, variables should be abstracted when used in multiple places, not just when they are asserted on. But, to answer your questions I was using these variables when starting the pending test. (I stopped to wait in some changes.) I'm fine removing the global variables for now since the context doesn't make sense.